routes/users.js

```diff
@@ -0,0 +1,12 @@
+const express = require('express');
+const router = express.Router();
+const Userscontroller = require('../controllers/users');
+
+router.get('/details', Userscontroller.index);
+router.post('/pushuser', Userscontroller.newUser);
```

Comment:
still proper route name not like pushuser it should be like createUser, for details use getusers

Reply:
done